homeassistant/components/prometheus/__init__.py

```diff
@@ -289,6 +294,19 @@ def _handle_climate(self, state):
             )
             metric.labels(**self._labels(state)).set(current_temp)
 
+        current_action = state.attributes.get(ATTR_HVAC_ACTION)
+        if current_action:
+            metric = self._metric(
+                "climate_action",
+                self.prometheus_cli.Gauge,
+                "Current HVAC action",
```

Comment:
The "Current" here should be removed as well.

Reply:
Correct, drop the "current". We should follow the naming of the attributes in Home Assistant, which is hvac_action.

Reply:
Removed "Current" from metric description.